pkgs/by-name/md/mdq/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mdq";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "yshavit";
+    repo = "mdq";
+    tag = "v${version}";
+    hash = "sha256-0KymmQ6uCViwB2TJ0Uj6dxaH26qbtvZ30hmzb+HwEuc=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-PJW4ByZ19jqQl6Yj07Shtnnd0x66BWz32A/P7oWUZbM=";
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "like jq but for Markdown: find specific elements in a md doc";
```

Comment:
```suggestion
    description = "JQ but for Markdown: find specific elements in a md doc";
```

Reply:
jq is stylized in all lowercase letters, capitalized "like"